src/common/hist_util.cc

```diff
@@ -2,14 +2,16 @@
  * Copyright 2017-2019 by Contributors
   * \file hist_util.cc
  */
-#include "./hist_util.h"
 #include <dmlc/timer.h>
-#include <rabit/rabit.h>
 #include <dmlc/omp.h>
+
+#include <rabit/rabit.h>
 #include <numeric>
 #include <vector>
 
+#include "xgboost/base.h"
 #include "../common/common.h"
+#include "./hist_util.h"
```

Comment:
This re-arrangement of headers can be avoided by including `version_config.h` in `base.h`.

Reply:
@hcho3 I will remove some re-arrangement of headers.  The `base.h` is included here for mm prefetch , not for version.